### PR TITLE
some initial docs pages

### PR DESCRIPTION
**Pull request by rphmeier**

### CODE REVIEW

File: docs-site/docs/intro.md [hunk 1]
```diff
@@ -2,4 +2,26 @@
 sidebar_position: 1
 ---
 
-# Intentionally Blank
\ No newline at end of file
+# Getting Started
+
+## Blobs
+
+The Blobs project exposes the **data availability** capabilities of [Polkadot](https://polkadot.network) and [Kusama](https://kusama.network) for general use. Use-cases include rollups and inscriptions.
```
Comment (pepyakin) [inline docs-site/docs/intro.md:9]: > inscriptions

this is going to age quick

File: docs-site/docs/node-operators/index.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Node Architecture
+
+Blobs is a standard Polkadot-SDK parachain node, which means that it actually runs two nodes:
+  1. A node for the Parachain (Blobs) on Polkadot or Kusama
+  2. A node for the Relay Chain, Polkadot or Kusama.
+
+You can pass arguments to each one of these underlying nodes with the following approach:
+
+```rust
```
Comment (pepyakin) [inline docs-site/docs/node-operators/index.md:18]: not really `rust`

File: docs-site/docs/node-operators/index.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  1. Running with `--blocks-pruning` on both 
+
+## Building From Source
+
+Building from source requires some familiarity with the terminal and your system's package manager.
+
+First, ensure you have an up-to-date Rust installation. If not, visit https://rustup.rs .
+
+Building from source requires a few packages to be installed on your system. On Debian or Ubuntu, these are:
+`libssl-dev protobuf-compiler cmake make pkg-config llvm clang`. On other linux distros or Mac, use your package manager's equivalents.
```
Comment (pepyakin) [inline docs-site/docs/node-operators/index.md:49]: prefer if those were on the new line in a separate block. The reasoning is when I personally skimming I am looking for those kind of commands as a block. Finding them in an inline block is tad bit harder.

File: docs-site/docs/intro.md (hunk 1 above)
Comment (rphmeier) [reply docs-site/docs/intro.md:9]: We'll leave it for now, but yeah, it will age quick :)

**Comment by rphmeier** [comment]
### Merge activity

* **Dec 27, 1:30 AM**: @rphmeier started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/thrumdev/blobs/146).

**Pull request merged**